app/human-detector-app/classes/Camera.tsx

```diff
@@ -6,54 +6,25 @@ import Group from './Group';
 
 export default class Camera {
   // field
-  cameraId: string;
+  private cameraId: string;
 
-  cameraName: string;
-
-  userId: string;
+  private cameraName: string;
 
   // constructor
-  constructor(userId: string, cameraName: string, cameraId: string) {
+  constructor(cameraName: string, cameraId: string) {
     this.cameraId = cameraId;
     this.cameraName = cameraName;
-    this.userId = userId;
   }
 
-  set setCameraName(cameraName: string) {
-    this.cameraName = cameraName;
+  get getCameraName() {
+    return this.cameraName;
   }
 
-  renameCamera(name: string): Camera {
-    // P1
-    return new Camera('0', 'name', 'ID');
+  get getCameraId() {
+    return this.cameraId;
```

Comment:
```suggestion
  get cameraId() {
    return this.cameraId;
```
Having `get` in the names for these getters is kinda `#wat`. If they're getters, you can access them like properties, so they should be named like properties. Otherwise, they should be regular methods